### ISSUE TITLE
773 $t

### ISSUE THREAD

**Issue by thefass** [ubtue/tuefind#1596]
Im Datensatz https://www.ixtheo.de/Record/1764253930#details wird in 773 $a angegeben, woraus der Aufsatz stammt

Im marc Datensatz in der WinIBW steht das in 773 $t:
_773 08$iEnthalten in$tProeven van vrouwenstudies theologie ; deel II:$dLeiden-Utrecht, 1991$g(1991), Seite 247-266$w(DE-627)1096539985$w(DE-576)026539985$z9064952450$7|||m_

Der Teil nach dem ;  (del II) wird dabei im Vufind nicht übernommen und auch nicht im Datensatz angezeigt. Dort steht nur:
![Bildschirmfoto 2021-07-28 um 16 08 31](https://user-images.githubusercontent.com/14169098/127336854-3a3a4b32-4395-4808-9e88-036cb6ebe238.png)

Wir wurden vom Autor der Reihe darauf aufmerksam gemacht, dass es wichtig wäre, wenn die Zählung mit angezeigt würde, also so:
Proeven van vrouwenstudies theologie ; deel II

PS: Warum ist in Vufind $a, was in WinIBW $t ist?

**Comment by mtrojan-ub** [comment]
Ich vermute stark, dass das in unserer Pipeline geändert wird. Daten vom BSZ sehen gut aus. 
@andreas-ub: Kannst Du bitte mal einen Blick darauf werfen? Ich vermute, dass die Ursache in augment_773a.cc liegt.

**Comment by andreas-ub** [comment]
Im Quellcode gibt es die Regel, dass $a gesetzt wird, wenn weder $a noch $t bei 773 existieren und $w mit (DE-627) vorhanden ist.

"deel II" ist vermutlich nicht enthalten,weil es in einem separaten Unterfeld ist ($n im 245 Feld des referenzierten Datensatzes)

marc_grep GesamtTiteldaten-210729.mrc 'if "001"=="1764253930" extract "773"' traditional

773 08$iEnthalten in$dLeiden-Utrecht, 1991$g(1991), Seite 247-266$w(DE-627)1096539985$w(DE-576)026539985$z9064952450$7|||m
773 18$gyear:1991$gpages:247-266 

marc_grep GesamtTiteldaten-210729.mrc 'if "001"=="1096539985" extract "245"' traditional

245 10$aProeven van vrouwenstudies theologie**$n**deel II / Onder red. van: Jonneke Bekkenkamp ..$cInteruniversitair Instituut voor Missiologie en Oecumenica. Onder redactie van: Jonneke Bekkenkamp ...

**Comment by obrandt** [comment]
Wir benötigen unbedingt einen Niederländischparser ;-)

**Comment by thefass** [comment]
ja, in 245 ist das nochmal drin, aber dort ist auch alles andere drin, was bei einer Kurztitelanzeige nicht angezeigt werden soll. Es geht speziell um die Zählung, so dass der Nutzer schnell sehen kann, aus welchem Band das stammt.

Vielleicht sollten wir due Umsetzung zurückstellen bis @IxTheoKm wieder aus dem Urlaub zurück ist, damit sie das nochmal beurteilen kann. Gewisse Felder werden ja aufgebaut in Voraussetzung von anderen Feldern.

**Comment by thefass** [comment]
Wenn die Regel lautet _Im Quellcode gibt es die Regel, dass $a gesetzt wird, wenn weder $a noch $t bei 773 existieren und $w mit (DE-627) vorhanden ist._ dann stellt sich die Frage noch dringlicher, warum in Vufind etwas $a ist, was im K10plus $t ist. Denn $t gibt es ja dort.

**Comment by thefass** [comment]
Wenn ich jetzt den Datensatz, so wie ich ihn in der WinIBW sehe mit dem, was @andreas-ub gepostet hat vergleiche, dann stelle ich fest, dass bei Euch $t in 773 gar nicht da ist. Seltsam. Kommt das bei Euch gar nicht an?
K10plus:
773 08$iEnthalten in**$tProeven van vrouwenstudies theologie ; deel II:**$dLeiden-Utrecht, 1991$g(1991), Seite 247-266$w(DE-627)1096539985$w(DE-576)026539985$z9064952450$7|||m

bei Euch:
773 08$iEnthalten in$dLeiden-Utrecht, 1991$g(1991), Seite 247-266$w(DE-627)1096539985$w(DE-576)026539985$z9064952450$7|||m

**Comment by mtrojan-ub** [comment]
Datensatz in BOSS:
https://swb.boss.bsz-bw.de/Record/(DE-627)1764253930#details

Dort gibt es ebenfalls kein 773t:
![grafik](https://user-images.githubusercontent.com/26873381/128018581-83a31fc4-2b70-441c-a455-5fd770eef52d.png)

**Comment by mtrojan-ub** [comment]
@thefass, wie gehen wir hier weiter vor? Möchtest Du die Frage bezüglich der fehlenden 773t-Unterfelder an den K10plus Support weiterleiten?

**Comment by relhei** [comment]
Würde es nicht ausreichen, wenn man bei der übergeordneten Aufnahme, d.h. dem Datensatz zum 2. Band des mehrteiligen Werks, 245 $n auswertet (d.h. auch anzeigt)? 
https://www.ixtheo.de/Record/1096539985#details
In der Volltitelanzeige des Aufsatzes ist ja der 2. Teil des mehrbändigen Werks richtig verlinkt.

**Comment by thefass** [comment]
ja, das ist eine gute Idee

**Comment by mtrojan-ub** [comment]
Das wird doch dort bereits angezeigt?

![grafik](https://user-images.githubusercontent.com/26873381/160585694-86153b79-579a-4853-9cee-81eb29c559f4.png)

![grafik](https://user-images.githubusercontent.com/26873381/160585806-5a4aa833-a1a4-4053-88ca-982806fe534d.png)

**Comment by relhei** [comment]
Ja, im Titel schon, aber nicht im Link. Der Link in der Titelanzeige eines Aufsatzes, der genau auf diesen Teil des mehrbändigen Werks führt (und nicht auf die Gesamtaufnahme des merhbändigen Werks), ist "abgekürzt":
![grafik](https://user-images.githubusercontent.com/25769591/160590744-1df249f0-c8e2-4705-8944-2471d65ecf24.png)

https://www.ixtheo.de/Record/1764253221

Die Linkbezeichnung sollte zusätzlich noch "/ deel II" enthalten, m. E. nicht aber die Herausgeberangabe. Was meint @thefass ?

**Comment by thefass** [comment]
> Die Linkbezeichnung sollte zusätzlich noch "/ deel II" enthalten, m. E. nicht aber die Herausgeberangabe.

genau

**Comment by mtrojan-ub** [comment]
Es handelt sich um eine 3-stufige Hierarchie:
1) [Proeven van vrouwenstudies theologie](https://www.ixtheo.de/Record/1090196644)
2) [Proeven van vrouwenstudies theologie / deel II / Onder red. van: Jonneke Bekkenkamp ..](https://www.ixtheo.de/Record/1096539985)
3) [Dr. Nicolette Bruining: de ontwikkeling van een "kritisch" theologe](https://www.ixtheo.de/Record/1764253221)

Wenn ich es richtig verstanden habe, sollen Informationen z.B. von 245an (übergeordnetes Werk) nach 773t (untergeordnetes Werk) übernommen werden. 

**Aber:** Wenn man eine Logik definiert, die Felder von 2) nach 3) übernimmt, dann hätte das ggf. auch zur Folge, dass Informationen von 1) nach 2) übernommen werden würden. Wie genau soll die Logik also lauten? Hier ein Beispiel bzw. Vorschlag:

Im Fall von 1) => 2) wäre 245an = "Proeven van vrouwenstudies theologie" schon das selbe wie das existierende 773a, daher wäre eine Übernahme in 773t nicht erforderlich.

Im Fall von 2) => 3) wäre 245an = "Proeven van vrouwenstudies theologie / deel II / Onder red. van: Jonneke Bekkenkamp ..". Da sich das von dem bestehenden 773a unterscheidet, würden wir hier ein 773t anlegen, richtig?

(Falls t existiert wird es beim Import bereits bevorzugt, a ist lediglich ein Fallback.)

Und soll das dann eigentlich nur für 773 gelten oder auch für 800, 810, 830? (=Schriftenreihen etc.)
